src/kARAN1595/StringEquality.java

```diff
@@ -0,0 +1,24 @@
+package kARAN1595;
+
+public class StringEquality {
+	int getRotationCount(String input1, String input2) {
+		int count = 0;
+		if (input1.length() == input2.length()) {
+			while (!input1.equals(input2)) {
```

Comment:
Just a suggestion. what if String can't be made equals even after all possible rotation ?? this will run infinitely ...

you can handled it by adding one more condition.
while(!input1.equals(input2) && count< input1.length)